### PR TITLE
Disable python profile-guided optimizations for Python-3.6.4-intel-2018a.eb

### PR DESCRIPTION
**Pull request by SimonPinches**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/p/Python/Python-3.6.4-intel-2018a.eb [hunk 1]
```diff
@@ -29,6 +29,8 @@ dependencies = [
 
 osdependencies = [('openssl-devel', 'libssl-dev', 'libopenssl-devel')]
 
+optimized = False
```
Comment (Micket) [inline easybuild/easyconfigs/p/Python/Python-3.6.4-intel-2018a.eb:32]: I think something like this definitely requires a motivating comment what issues it causes.

**Review by Micket — CHANGES_REQUESTED** [review]
So, how "bad" is this, is it some precision issue with intel compilers only?
Does it only affect 3.6 or intel-2018a?

I'm also a bit surprised to see this option, i would have thought we would use toolchainopts parameters for this?

File: easybuild/easyconfigs/p/Python/Python-3.6.4-intel-2018a.eb (hunk 1 above)
Comment (SimonPinches) [reply easybuild/easyconfigs/p/Python/Python-3.6.4-intel-2018a.eb:32]: I think this PR can be dropped in favour of [this easyblock one](https://github.com/easybuilders/easybuild-easyblocks/pull/2228).  I only experienced problems with `intel/2018a` but from what I gather, I was not alone.  I also agree that I should have added a motivating comment, next time...

**Comment by Micket** [comment]
Since the easyblock fix was merged, I will close this!

**Pull request closed without merging**